spec/models/league_spec.rb

```diff
@@ -140,6 +141,146 @@
         expect(subject.length).to eq(league.teams.length * (league.teams.length - 1))
       end
     end
+
+    context 'swiss system' do
+      let(:gamemode) { League.game_modes[:swiss] }
+
+      it 'does not have an up-to-date schedule if the last gameday is in the past' do
+        league.gamedays << FactoryBot.create(:gameday)
+
+        last_gameday = league.gamedays.last
+        last_gameday.starttime = 2.days.ago
```

Comment:
another way of specifying dates? :D Yeh, maybe we should have a talk about this :P

Reply:
I think this is the most readable syntax, why not?

Reply:
I just never know which versions are timezone dependent and which aren't so if we mix them up, they'll lead to issues at some point.